docs/operator/operations.md

```diff
@@ -0,0 +1,100 @@
+| Work Queue Processing Latency             | Indicates how long in seconds an item stays in workqueue before being requested.                                                                     |
+| Work Queue Processing Duration            | Indicates how long in seconds processing an item from workqueue takes.                                                                               |
+| Work Queue Depth                          | Indicates the number of actions waiting in the queue to be performed.                                                                                |
+| Work Queue Add Rate                       | Indicates the rate of adding actions to the queue.                                                                                                   |
+| Self-signed Certificate Not Renew         | Indicates the self-signed Certificate of the related Kyma CR has not been renewed yet.                                                                            |
+| Purge Duration Seconds                    | Shows the duration of the purge reconciliation.                                                                                                      |
+| Purge Count                               | Shows the total number of purges.                                                                                                                    |
+| Purge Errors                              | Shows the errors produced by the purge.                                                                                                              |
+
+
+#### 2. Kyma CRs Status
+This dashboard gives an overview of the Kyma CRs and modules states. It includes the following:
+
+| Panel Name                    | Description                                                                                     |
+|-------------------------------|-------------------------------------------------------------------------------------------------|
+| Enabled Modules               | Shows the number of all modules enabled in the SKR Clusters, including the total number for every module. |
+| Kyma Manifest in Error State  | Shows the Manifest CRs in the `Error` state with the information about the relevant Kyma CR name, module name, and shoot name. |
+| Kyma State Total              | A graph showing the number of Kyma CRs in each state over time.                                  |
+| Module State Total            | A graph showing the number of modules added to Kyma CRs in each state over time.               |
+| Mandatory Module State Total  | A graph showing the number of mandatory modules added to Kyma CRs in each state over time.     |
+| Mandatory Modules Count       | Indicates the number of mandatory modules installed in the SKR clusters.                       |
+
+
+#### 3. Runtime Watcher Components
+This dashboard gives an overview of the Runtime Watcher components. It includes the following:
```

Comment:
As I do not see any other docs, from where does the reader ought to know what the "watcher" is, what is its function? So far, for the reader there's a name, without a meaning and a function.